11.0/Dockerfile

```diff
@@ -0,0 +1,113 @@
+FROM debian:jessie
+MAINTAINER Camptocamp
+
+# create the working directory and a place to set the logs (if wanted)
+RUN mkdir -p /opt/odoo /var/log/odoo
+
+WORKDIR "/opt/odoo"
+
+COPY ./base_requirements.txt ./
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && apt-get update \
+        && apt-get install -y --no-install-recommends \
+            antiword \
+            ca-certificates \
+            curl \
+            ghostscript \
+            graphviz \
+            less \
+            nano \
+            node-clean-css \
+            node-less \
+            poppler-utils \
+            python \
+            python-libxslt1 \
+            python-pip \
+            xfonts-75dpi \
+            xfonts-base \
+            # build packages to clean after the pip install
+            build-essential \
+            python-dev \
+            libfreetype6-dev \
+            libxml2-dev \
+            libxslt1-dev \
+            libsasl2-dev \
+            libldap2-dev \
+            libssl-dev \
+            libjpeg-dev \
+            zlib1g-dev \
+            libfreetype6-dev \
+            tcl expect \
+        && curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+        && apt-get update \
+        && apt-get install -y --no-install-recommends \
+            postgresql-client \
+            libpq-dev \
+        && curl -o wkhtmltox.deb -SL http://nightly.odoo.com/extra/wkhtmltox-0.12.1.2_linux-jessie-amd64.deb \
```

Comment:
this one might be troublesome for stretch version, as far as I know there is no stretch package proposed by Odoo yet

Reply:
According to the official odoo docker image for v11.0 wkhtmltopdf is taken from here:

https://github.com/wkhtmltopdf/wkhtmltopdf/releases/tag/0.12.4

ref: https://github.com/odoo/docker/blob/60b3f82cf1ccb05878b6ce7b470ea5ce80753745/11.0/Dockerfile#L16